web/pdf_page_view.js

```diff
@@ -697,6 +698,7 @@ class PDFPageView {
 
     let textLayer = null;
     if (this.textLayerMode !== TextLayerMode.DISABLE && this.textLayerFactory) {
+      this._accessibilityManager = new TextAccessibilityManager(div);
```

Comment:
Given how the textLayer is being re-created on zooming/rotation, but the annotationLayer and annotationEditorLayer are kept, will this work correctly as-is?
With this code the annotationLayer/annotationEditorLayer will potentially contain a reference to an "old" `TextAccessibilityManager`, which seems like it probably doesn't work.

Can we just re-use the same `TextAccessibilityManager`-instance, but still lazily initialized, with the following code?
```suggestion
      this._accessibilityManager ||= new TextAccessibilityManager(div);
```

Reply:
I found this bug.